src/draw/DefaultDrawer.js

```diff
@@ -193,66 +210,41 @@ class DefaultDrawer {
   }
 
   /**
-   * Initialize all draw option of the components.
+   * Initializes all draw option of the components.
+   * @param {Object} d3Element - D3 selected elements.
    * @param {Component[]} components - List of Components.
-   * @param {Object} d3Element - D3 selected element.
+   * @param {String} parentId - ID of parent it contains components.
    */
-  initializeComponents(components, d3Element) {
-    let componentsToInit = components.filter((component) => {
+  initializeComponents(d3Elements, components, parentId) {
```

Comment:
`d3Elements` not accordingly to the description of the method.

Reply:
Done